src/org/zaproxy/zap/extension/fuzz/httpfuzzer/ui/HttpFuzzResultsContentPanel.java

```diff
@@ -116,6 +126,37 @@ public void itemStateChanged(ItemEvent e) {
 
         toolbar.add(Box.createHorizontalStrut(16));
         toolbar.add(showErrorsToggleButton);
+        
+        
+        JButton button = new JButton((Constant.messages.getString("fuzz.httpfuzzer.results.toolbar.button.export")));
+        button.setIcon(new ImageIcon(HttpFuzzResultsContentPanel.class.getResource("/resource/icon/16/115.png")));
+        button.addActionListener((new AbstractAction() {
+			private static final long serialVersionUID = 1L;
+
+			@Override
+			public void actionPerformed(ActionEvent e) {
+				WritableFileChooser chooser = new WritableFileChooser();
+				chooser.setSelectedFile(new File(Constant.messages.getString("fuzz.httpfuzzer.results.toolbar.button.export.defaultName")));
+				if (chooser.showSaveDialog(null) == WritableFileChooser.APPROVE_OPTION) {
+					String file = chooser.getSelectedFile().toString();
+					if (!file.endsWith(".csv")) file += ".csv";
+					try (CSVPrinter pw = new CSVPrinter(new PrintWriter(new File(file)), CSVFormat.newFormat(',').withRecordSeparator("\n"))) {

```

Comment:
Any reason to use the PrintWriter? Why not a BufferedWriter (e.g. Files.newBufferedWriter(...))?

If there's no quote character the commas of the fuzz values will be interpreted as values of other columns. Why not use CSVFormat.DEFAULT instead?